src/WebJobs.Script/Config/ScriptHostConfiguration.cs

```diff
@@ -40,6 +41,12 @@ public ScriptHostConfiguration()
         public string RootLogPath { get; set; }
 
         /// <summary>
+        /// Gets or sets the root path to search for binding
+        /// extensions.
+        /// </summary>
+        public string RootExtensionsPath { get; set; }
```

Comment:
Added a first class config setting for this, just like we have for all our other path properties